client/src/deploy/send.rs

```diff
@@ -0,0 +1,46 @@
+use clap::{App, ArgMatches, SubCommand};
+
+use casper_node::types::Deploy;
+
+use super::creation_common;
+use crate::{command::ClientCommand, common, RpcClient};
+
+pub struct SendDeploy;
+
+// TODO: rename casper_node account type PutDeploy => SendDeploy, and account_send_deploy
+impl RpcClient for SendDeploy {
+    const RPC_METHOD: &'static str = "account_put_deploy";
+}
+
+impl<'a, 'b> ClientCommand<'a, 'b> for SendDeploy {
+    const NAME: &'static str = "send-deploy";
+    const ABOUT: &'static str =
+        "Deploy a smart contract file to the Casper network via an existing running node.";
+
+    fn build(display_order: usize) -> App<'a, 'b> {
+        let subcommand = SubCommand::with_name(Self::NAME)
+            .about(Self::ABOUT)
+            .display_order(display_order)
+            .arg(creation_common::input::arg(display_order));
+        creation_common::apply_common_creation_options(subcommand)
+    }
+
+    fn run(matches: &ArgMatches<'_>) {
+        creation_common::show_arg_examples_and_exit_if_required(matches);
+
+        let node_address = common::node_address::get(matches);
+
+        let input_path = creation_common::input::get(matches);
+        let input = std::fs::read(&input_path)
```

Comment:
I'd expect reading in a deploy will be the same for `sign-deploy` and `send-deploy`, so I think that functionality should live in `creation_common` rather than being duplicated.  (Makes `creation_common` poorly named, since this isn't really to do with creation of a Deploy.  If you have a better idea for that module's name, feel free.  But just `common` would be confusing with the existing `common` module one level up).

Reply:
This is the kind of thing I would put in the mod.rs - perhaps deploy.rs higher up?

Reply:
Yup - that probably makes more sense than putting it in `creation_common`.